core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -420,4 +421,21 @@ public int getBip32HeaderPriv() {
      * networks.
      */
     public abstract boolean hasMaxMoney();
+
+    /**
+     * Return the default serializer for this network. This is a shared serializer.
+     * @return 
+     */
+    public final MessageSerializer getDefaultSerializer() {
```

Comment:
In theory it's fine without, as creating two serializers isn't the end of the world, realised should just in case someone makes the serializer mutable.